### Write a Program to iterate over each letter, and print it from the word : `Welcome`

In [1]:
word = "Welcome"
for char in word :
    print(char)

W
e
l
c
o
m
e


In [2]:
for char in "Welcome" :
    print(char)

W
e
l
c
o
m
e


### `list_1 = [10,20,30,40,50,"hello","world"]`

In [4]:
list_1 = [10,20,30,40,50,"hello","world"]
for cat in list_1:
    print(cat)

10
20
30
40
50
hello
world


### Create a Python list with numbers,and perform the sum of all elements init.

In [4]:
py_list1 = [10,20,30,40,50]
sum_total = 0
for x in py_list1:
    sum_total += x
print("Total Sum is: ",sum_total)

Total Sum is:  150


### Nest one loop inside another

In [5]:
py_list1

[10, 20, 30, 40, 50]

In [5]:
py_list2 = [50,40,30,20,10]
for x in py_list1:
    print("My value of x is : ",x)
    for y in py_list2:
        print("My Value of y is:",y,"My value of X is:",x)

NameError: name 'py_list1' is not defined

# Activity

Create a Python list as :

`list1 = [12, 15, 32, 42, 55, 75, 122, 132, 150, 180, 200]`

iterate over the list and print each element, and compute the sum & average